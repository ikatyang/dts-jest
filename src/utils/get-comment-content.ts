export const get_comment_content = (comment: string) =>
  comment.startsWith('//')
    ? comment.slice(2).trim()
    : comment.slice(2, -2).trim();
