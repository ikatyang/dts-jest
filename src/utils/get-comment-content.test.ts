import { get_comment_content } from './get-comment-content';

it('should return correctly for single line comment', () => {
  expect(get_comment_content('// single-line-comment')).toMatchSnapshot();
});

it('should return correctly for multi line comment', () => {
  expect(get_comment_content('/* multi-line-comment */')).toMatchSnapshot();
});
