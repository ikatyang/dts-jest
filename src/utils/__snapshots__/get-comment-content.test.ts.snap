// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return correctly for multi line comment 1`] = `"multi-line-comment"`;

exports[`should return correctly for single line comment 1`] = `"single-line-comment"`;
